incuna-sass/legacy/_box-sizing.sass

```diff
@@ -1,14 +1,14 @@
-%border-box
+%legacy-border-box
```

Comment:
the general idea was to avoid conflicts if we're using both bourbon and the legacy folder. I know not all of these are in bourbon but it seemed more consistent to make everything `legacy-` than picking and choosing